Rakefile

```diff
@@ -23,7 +23,9 @@ end
 
 desc 'Compile the SVG icons into the font'
 task :compile do
-  system 'fontcustom compile'
+  result = `fontcustom compile`
+  puts result
+  raise StandardError if result =~ /error/
```

Comment:
If it exits non-zero on failure, then you can just do

```ruby
exit $CHILD_STATUS.exitstatus unless system('fontcustom compile')
```

This will write to the screen and exit with the same exit code, which should fail in the same way.